sdk/core/azure-core/inc/azure/core/datetime.hpp

```diff
@@ -113,107 +247,68 @@ namespace Azure { namespace Core {
      *
      * @throw DateTimeException If year exceeds 9999, or if \p format is not recognized.
      */
-    std::string ToString(DateFormat format = DateFormat::Rfc1123) const
+    std::string GetString(DateFormat format) const
     {
-      return ToString(format, TimeFractionFormat::DropTrailingZeros);
+      return GetString(format, TimeFractionFormat::DropTrailingZeros);
     };
 
     /**
-     * @brief Get a string representation of the @DateTime formated with ISO8601.
+     * @brief Get a string representation of the @DateTime formatted with ISO 8601.
      *
-     * @param fractionFormat The format that is applied to the fraction part from the ISO8601 date.
+     * @param fractionFormat The format that is applied to the fraction part from the ISO 8601 date.
      *
      * @throw DateTimeException If year exceeds 9999, or if \p fractionFormat is not recognized.
      */
-    std::string ToIso8601String(TimeFractionFormat fractionFormat) const
+    std::string GetIso8601String(TimeFractionFormat fractionFormat) const
```

Comment:
This should be an overload of GetString with a DateFormat

Reply:
I provided similar feedback here: https://github.com/Azure/azure-sdk-for-cpp/pull/936#discussion_r520269392

Which is why this seems out of the scope of the PR.

Reply:
RFC 1123 does not have fractional seconds / TimeFractionFormat has no effect with it, so there are two methods: `GetString(DateTimeFormat)`, and `GetIso8601String(TimeFractionFormat)`.